lightspeed/modules/proc_monitor-onpremise-deployment.adoc

```diff
@@ -2,25 +2,37 @@
-
-.Prerequisites
-* You have set up a {LightspeedShortName} on-premise deployment.
+After the {LightspeedShortName} on-premise deployment is successful, use the following procedure to monitor the metrics on an API endpoint `/metrics`. 
 
 .Procedure
 
 . Create a *system auditor* user:
-.. Create a user with the *system auditor* role in the the {PlatformName}. Refer to the procedure in the topic link:{BaseURL}/red_hat_ansible_automation_platform/{PlatformVers}/html-single/automation_controller_user_guide/index#proc-controller-creating-a-user[Creating a user] in the _Automation Controller User Guide_.
+.. Create a user with a *system auditor* role in the the {PlatformName}. Refer to the procedure in the topic link:{BaseURL}/red_hat_ansible_automation_platform/{PlatformVers}/html-single/automation_controller_user_guide/index#proc-controller-creating-a-user[Creating a user] in the _Automation Controller User Guide_.
 
-.. Verify that you can log in to the link:https://c.ai.ansible.redhat.com/[Ansible Lightspeed portal login page], and then log out.
+.. Verify that you can log in to the Ansible Lightspeed portal for on-premise deployment (`\https://<lightspeed_route>/`) as the newly-created system auditor user, and then log out.
 
 . Create a token for the *system auditor* user:
-.. Log in to the link:https://c.ai.ansible.redhat.com/[Ansible Lightspeed portal login page] as an organization administrator.
-.. Create a token for the *system auditor* user. Refer to the procedure in the topic link:{BaseURL}/red_hat_ansible_automation_platform/{PlatformVers}/html-single/automation_controller_user_guide/index#proc-controller-user-tokens[Creating tokens for a user] in the _Automation Controller User Guide_.
-.. Log out from the portal. 
-
-. Monitor your {LightspeedShortName} on-premise deployment:
-.. Log in to the link:https://c.ai.ansible.redhat.com/[Ansible Lightspeed portal login page] as the system auditor user.
-.. Use the Authorization token to access the metrics endpoint `/api/v2/metrics/`. Refer to the procedure in the topic link:{BaseURL}/red_hat_ansible_automation_platform/{PlatformVers}/html-single/automation_controller_administration_guide/index#assembly-controller-metrics[Metrics] in the _Automation Controller Administration Guide_.
\ No newline at end of file
+.. Log in to the Ansible Lightspeed portal for on-premise deployment (`\https://<lightspeed_route>/admin`) as an administrator by using the following credentials:
++
+* Username: *admin*
+* Password: The secret that is named as `<lightspeed-custom-resource-name>-admin-password` in the {OCP} cluster namespace where {LightspeedShortName} is deployed. 
+
+.. On the Django administration window, select *Users* from the Users area. A list of users is displayed.
+.. Verify that the user with the system auditor role is listed in the *Users* list.
+.. From the Django Oauth toolkit area, select menu:Access tokens [Add].
```

Comment:
Remove space in the menu macro or it will format incorrectly:

menu:Access tokens[Add]

Reply:
Done, thank you